artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -350,6 +352,8 @@ private String appendIgnoresToFilter(String filterString) {
       filterString += "!" + storeAndForwardPrefix;
       filterString += ",!" + managementAddress;
       filterString += ",!" + managementNotificationAddress;
+      //advisory topics shouldn't be clustered
+      filterString += ",!" + ADVISORY_TOPIC_PREFIX;
```

Comment:
This is Openwire specific, should avoid pushing protocol specifics into core.

What happens it another protocol needs an address not clustered?

Would be better to design this in an agnostic way, eg a flag on the address or queue, to not cluster bridge, which the protocol managers can set on creating

Reply:
The best wat I could think is to inject it as a constructor parameter similar to managementNotificationAddress; but advisory topics are just an OpenWire specific feature anyway